sources/packages/backend/apps/api/src/route-controllers/assessment/_tests_/e2e/assessment.aest.controller.manualReassessment.e2e-spec.ts

```diff
@@ -0,0 +1,236 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentAppeal,
+  createFakeStudentAppealRequest,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import {
+  ApplicationStatus,
+  AssessmentTriggerType,
+  NoteType,
+  StudentAssessmentStatus,
+} from "@sims/sims-db";
+
+describe("AssessmentAESTController(e2e)-manualReassessment", () => {
+  let app: INestApplication;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+  });
+
+  it("Should create a manual reassessment when requested.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource);
+    application.currentAssessment.studentAssessmentStatus =
+      StudentAssessmentStatus.Completed;
+    application.studentAssessments = [application.currentAssessment];
```

Comment:
I would say that this piece of code can move into the factory `saveFakeApplication`

Reply:
It seems a good idea but as we tried to do so and breaking other tests because of the change in the factory, we're not doing it now.

Reply:
@andrewsignori-aot @guru-aot @sh16011993 @katrina-aot-git @lewischen-aot 

Updating the factory was breaking the follwing tests in queue consumers.

![image](https://github.com/bcgov/SIMS/assets/54600590/17e8b514-cdf5-467e-923b-7c9105312903)

with following reason 

![image](https://github.com/bcgov/SIMS/assets/54600590/3158f297-e609-4a43-b24b-8a21e30d5708)

Reply:
The code can be moved and the error should be due to something else.
I would not consider it a blocker right now.